sample-apps/ruby-rails-sample-app/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,78 @@
+require 'net/http'
+
+def get_xray_trace_id(otel_trace_id_hex)
```

Comment:
Can you add some documentation about how this conversion works? 

Here is how to add documentation https://github.com/ruby/rdoc#label-Writing+Documentation

In the documentation string, please add a link to a reference about xray trace ids.

Reply:
Added documentation